docs/3.0rc/cloud/self-host.mdx

```diff
@@ -1,16 +1,66 @@
 ---
 title: Host a Prefect server instance
-description: Hosting Prefect Sever
 ---
 
-
 <Note>
-    If you would like to host a Prefect server instance on Kubernetes, check out the prefect-server [Helm chart](https://github.com/PrefectHQ/prefect-helm/tree/main/charts/prefect-server).
+If you would like to host a Prefect server instance on Kubernetes, check out the prefect-server [Helm chart](https://github.com/PrefectHQ/prefect-helm/tree/main/charts/prefect-server).
 </Note>
 
-After installing Prefect, you have a Python SDK client that can communicate with [Prefect Cloud](https://app.prefect.cloud), and an [API server](/3.0rc/api-ref/) instance backed by a database and a UI.
+After installing Prefect, you have a Python SDK client that can communicate with either [Prefect Cloud](/3.0rc/cloud/) or a self-hosted Prefect server, backed by a database and a UI.
+
+Prefect Cloud and a self-hosted Prefect server instance share a common set of capabilities. Prefect Cloud has these additional features:
+
+- [Workspaces](/3.0rc/cloud/workspaces/)&mdash;isolated environments to organize your flows, deployments, and flow runs
+- [Automations](/3.0rc/react/understand-events/automations-triggers/)&mdash;configure triggers, actions, and notifications in response to real-time monitoring events
+- [Email notifications](/3.0rc/react/understand-events/automations-triggers/)&mdash;send email alerts from Prefect's servers based on automation triggers
+- [Service accounts](/3.0rc/cloud/manage-users/service-accounts/)&mdash;configure API access for running workers or executing flow runs on remote infrastructure
+- [Custom role-based access controls (RBAC)](/3.0rc/cloud/manage-users/manage-roles/)&mdash;assign users granular permissions to perform activities within an account or workspace
+- [Single Sign-on (SSO)](/3.0rc/cloud/manage-users/configure-sso/)&mdash;authentication using your identity provider
+- [Audit Log](/3.0rc/cloud/manage-users/audit-logs/)&mdash;a record of user activities to monitor security and compliance
+
+## Prefect server installation notes
+
+Your self-hosted server must meet certain requirements and be properly configured. 
+
+### SQLite
+
+Hosting your own Prefect server instance requires a PostgreSQL database, but you can use SQLite 2.24 or newer for development. 
```

Comment:
Just noting that we might change the guidance here. @billpalombi is up on the latest thinking. Don't let potential changes here hold back merging.

Reply:
Great callout. We can remove this line entirely.